edgelet/iotedge/src/lib.rs

```diff
@@ -48,5 +48,5 @@ pub trait Command {
 
 #[derive(Debug, Deserialize)]
 pub struct LatestVersions {
-    pub aziot_edged: String,
+    pub iotedged: String,
```

Comment:
Why?

Reply:
@massand seems to have changed it as part of his PR (probably as part of a larger search + replace).

The actual JSON blob still uses `iotedge` as the key, so using `aziot_edged` isn't valid.

Reply:
You're making the commit to master, not to 1.1. The "actual JSON blob" will use `"aziot-edged"` when it's released.

Reply:
Actually, on that note, we should change the URL too, to `https://aka.ms/latest-aziot-edge-stable`

@veyalla, @micahl: We don't want `iotedge check` on 1.1 to think 1.2 is a newer version, right? So `latest-iotedge-stable` will say `"1.1"` even after 1.2 is released?

Reply:
Ah, fair enough. I guess I'd just seen that the check was failing, and thought to quickly "fix" it alongside these other changes.

But yes, this does bring up a good point that we should have some way of ensuring that we don't break existing code by renaming the JSON field.

Reply:
@arsing I think we should leave this change in for now, since updating this field name will have to be coordinated with any changes to the aka.ms URL + JSON structure in the lead-up to 1.2. 

Moreover, at the moment, leaving the field as "aziot_edged" is plain wrong on two accounts:
a) the current aka.ms URL doesn't match the JSON, and 
b) the JSON field will eventually be `"aziot-edged"`, not `"aziot_edged"` as it is currently.

I'd like to merge this PR to fix the particular issue at hand, and open another one in the future after the PMs have decided how we want to tackle the latest-version JSON question.